assets/js/app.js

```diff
@@ -0,0 +1,4 @@
+document.getElementById("switchlang").addEventListener("click", function() {
+  const localization = document.getElementById("switchlang");
```

Comment:
1. don't use anonymous functions for event handling
2. get your checkbox inside handler via event